### ISSUE TITLE
A11y_AzureDataStudio_SaveJupyterBookPreviewTab_ScreenReader: NVDA is not announcing the order for the list item under the "Chapter" heading .

### ISSUE THREAD

**Issue by 12-shweta** [microsoft/azuredatastudio#7792]
[Check out Accessibility Insights!](https://accessibilityinsights.io/)- Identify accessibility bugs before check-in and make bug fixing faster and easier.”

GitHubTags:#A11y_AzureDataStudio;#A11yMAS;#A11yTCS;#WCAG1.3.1;#DesktopApp;#JAWS;#NVDA;#SQL Azure Data Studio;#Win32

**Environment Details:	**
Application Name: Azure Data Studio
Version: 1.12.0-insider (user setup)
Commit:6582debd7388b1be512fc3a9b956d3e54f6115e7
Date: 2019-10-03T08:56:27.454Z
VS Code: 1.38.0
Electron: 4.2.10
Chrome: 69.0.3497.128
Node.js: 10.11.0
V8: 6.9.427.31-electron.0
OS: Windows_NT x64 10.0.18362
NVDA Version: 2019.2
JAWS Version:2019.1907.42 ILM

**Additional Details:**
Tool Name : NVDA,JAWS

**Steps to Reproduce:**
1.Install and Launch the application the azure data studio application.
2.Connect to the servers with the required credentials.
3.Navigate to the control panel and click on the "jupyter books" icon.
4.Press CRTL+SHIFT+P and search "jupyter books: SQL Server 2019 Guide" 
5.Navigate to the left section of jupyter books under the untitled books and click on "save" icon .
6.Navigate to the saved books and click on to the "Home". 
7.Start NVDA.
8.Navigate till “Monitor Kubernetes” using tab.
9.Observe the announcement by screen reader .
 
**Actual:**
Screen reader is only announcing the “Monitor Kubernetes” with its value and not announcing the order for that list item.

**Expected:** 
Screen reader  should announce the order for the list item like “Monitor Kubernetes 1 of 8” with its value.

**User Impact:**
Screen reader user will it difficult to understand the order of list item while navigating only through tab.

**Attachment for Reference:**
![7792_A11y_AzureDataStudio_SaveJupyterBookPreviewTab_ScreenReader](https://user-images.githubusercontent.com/54656883/67071620-a235b180-f1a0-11e9-904d-793bdac0f56c.png)

Does this issue occur when all extensions are disabled?: Yes

**Comment by chlafreniere** [comment]
@12-shweta i can't repro this when scanning through the page using nvda. here's what i see:

![image](https://user-images.githubusercontent.com/40371649/70488594-0d6b7800-1aae-11ea-883a-9e612c6eda39.png)

Can you verify that the issue persists on your side?

**Comment by chlafreniere** [comment]
Per further guidance, closing this for now. @12-shweta if you can still repro feel free to reopen.

**Comment by 12-shweta** [comment]
Verified the bug in below environment :
Version: 1.15.0-insider (user setup)
Commit: 041f34bedaa4f08ba5e7b2a28dc4bb0be90a2462
Date: 2020-01-08T05:36:19.161Z
VS Code: 1.38.0
Electron: 6.1.6
Chrome: 76.0.3809.146
Node.js: 12.4.0
V8: 7.6.303.31-electron.0
OS: Windows_NT x64 10.0.18363
Issue is fixed and does not reproduce .
![image](https://user-images.githubusercontent.com/54656883/72051255-e42a4b80-32e8-11ea-9d74-a0d38a858b3c.png)

**Comment by fsteffi** [comment]
#Azure Data Studio - New Features_win32;

**Comment by Rama8345** [comment]
A11ySev2

**Comment by Samhitha66** [comment]
#Closed; #Regressed:07-14-23;
